### PR TITLE
[19904] Datepicker admin settings

### PR DESCRIPTION
**Pull request by floriank**
This PR supersedes @hirtie-maxim's #2980 - the other datepickers broke with his solution.

Meets the requirements of https://community.openproject.org/work_packages/19904.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://floriank-fix-19904-datepicker-admin-settings.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://floriank-fix-19904-datepicker-admin-settings.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -504,7 +504,8 @@ def include_calendar_headers_tags
         else
           '' # use language
         end
-        js = "var CS = { lang: '#{current_language.to_s.downcase}', firstWeekDay: '#{start_of_week}' };"
+        # FIXME: Get rid of this abomination
```
Comment (myabc) [inline app/helpers/application_helper.rb:507]: :+1: for this.

**Comment by floriank** [comment]
@opf/developers with no comment on this, I would like to merge this.

**Comment by myabc** [comment]
LGTM code-wise

**Pull request merged**